add initial set of REQ- requirements (#9)
SpencerDawkins/net-collab-rqmts: draft-kwbdgrr-tsvwg-net-collab-rqmts.md

These are necessary to rebuild receiver state after loss of delta frames.
The key frames are therefore more critical to deliver to the receiver than delta frames.

Streaming video also contains audio frames which can be encoded

> separately and thus can be signaled separately (requirement:

Reviewer: ```suggestion
separately and, thus, can be signaled separately (requirement:
```

REQ-MEDIA-KEYFRAME).

> Audio is more critical than video for almost all applications, but its

Reviewer: I would reword this as this is not inclusive. Video or signals are important for persons with inabilities. Refer to Total Conversation concept.

Author: new wording is 'many applications', and many (most) people can hear.  We are well above 80/20 threshold of the common case that audio is more critical than video, most of the time.  The ability of the receiver to change the priority by communicating to the ISP -- without cooperation of the sender -- is a requirement and gives a hearing-impaired user the ability to adjust video above audio.

Reviewer: Agreed, especially the last part of your reply.

> Audio is more critical than video for almost all applications, but its

Reviewer: ```suggestion
Audio is more critical than video for many applications, but its
```

importance (relative to other packets in the flow) is still an
application decision (requirements: REQ-MEDIA-AV-SEPARATE,
REQ-MEDIA-CLIENT-DECIDES).

Especially with media over QUIC, the server or proxy sends the same
stream to all receivers, including the same metadata.  Thus, when a
client needs different prioritization (e.g., video over audio), this
is only achievable by signaling that priority inversion from the

> client to the ISP router (requirement: REQ-CLIENT-DECIDES).

Reviewer: Does this assume the host is always directly connected to the ISP network?

Author: No, it assumes the client can tell the ISP router the priority of the incoming flow as the client wants.  The client could be behind a NAT.

Reviewer: I would zoom into that part to avoid confusion.  We can reason about "upstream router (home router, ISP router, etc.)" to cover both direct and indirect cases

Examples: live broadcast, on-demand video streaming

REQ-MEDIA-KEYFRAME: Video contains partial frames and full frames, which need to be distinguished so that
full frames can be indicated to the network.

REQ-MEDIA-AV-SEPARATE:  Audio can be prioritized differently than video.

## Interactive Media {#uc-interactive}

Interactive media includes content that a user can actively engage with and results in input and response actions that can be highly delay sensitive. It also includes mixed traffic where both bulk and interactive data are exchanged within the same flow.

> They may include digital models of the real world, multimedia content, virtualized desktop/apps and interactive engagement.

Reviewer: ```suggestion
They may include digital models of the real world, multimedia content, virtualized desktop/apps, and interactive engagement.
```

> Examples: VoIP (peer-to-peer (P2P), group conferencing), gaming, eXtended Reality (XR), Citrix Virtual Apps and Desktops.

Reviewer: ```suggestion
Examples: VoIP (Peer-to-Peer (P2P), group conferencing), gaming, eXtended Reality (XR), Virtual Apps and Desktops.
```

No need to mention a product name :-)

> REQ-INTERACTIVE: The importance within interactive traffic flow depends on the type of application and the user activity. Requirements for different interactive media is listed below:

Reviewer: ```suggestion
REQ-INTERACTIVE: The importance within interactive traffic flow depends on the type of application and the user activity. Requirements for different interactive media are listed below:
```

3. Mixed Traffic: Signal flow will vary depending on the nature of the packet. With variety of traffic going through the session, some packets can contain interactive traffic while the others contain bulk transfer. There can be combination of reliable and unreliable traffic within the same session through multiple streams. Client-to-network signaling plays a vital role in effectively forwarding mixed traffic for better user interactivity and network performance.

> ## User preferences {#uc-preferences}

Reviewer: ```suggestion
## User Preferences {#uc-preferences}
```
